### ISSUE TITLE
ECMWF acquisition crash

### ISSUE THREAD

**Issue by oceancolorcoder** [nasa/HyperCP#265]
Execution halts when unable to acquire ECMWF model.

Test file:
PACE-PAX_PACE-PAX_20240912_160000_L1AQC on the Box'

```
ECMWF data for Wind and AOD may be used to replace blank values. Reading in model data...
Nearest model found at 2024-09-12T12:00:00+00:00
2024-10-26 01:41:45,870 INFO Sending request to https://ads.atmosphere.copernicus.eu/api/v2/resources/cams-global-atmospheric-composition-forecasts
INFO:cdsapi:Sending request to https://ads.atmosphere.copernicus.eu/api/v2/resources/cams-global-atmospheric-composition-forecasts
EAC4 atmospheric data could not be retrieved. Check inputs.
```

**Comment by juanchossn** [comment]
Should be fixed after #278

**Comment by oceancolorcoder** [comment]
resolved in v1.2.10
